Makefile

```diff
@@ -104,29 +104,34 @@ e2e.test:
 test-e2e: e2e.test
 	./e2e.test -test.timeout 180m -test.v -ginkgo.v
 
-test-go: generate
-	go build -tags containers_image_openpgp ./...
+test-go:
```

Comment:
are not we missing the dependency on generate here ?

Reply:
Yes we are and we are not, I am still figuring out how to piece it best. I would like us to call generate mindfully not implicitly during `make test`. I am now waiting for the ci and will iterate.